examples/getting_started/run.py

```diff
@@ -42,23 +134,28 @@
     for i in range(NUM_STEPS):
         if config.SIMULATION_CONTROLLER.AGENT == "SphereAgent":
             obs, reward, done, info = env.step(action={
-                "add_force": np.array([5000.0, 0.0, 0.0], dtype=np.float64),
-                "add_rotation": np.array([0.0, 1.0, 0.0], dtype=np.float64)
+                "add_force": np.array([10000.0, 0.0, 0.0], dtype=np.float64),
+                "add_rotation": np.array([0.0, 0.0, 1.0])
             })
             if not args.benchmark:
-                spear.log("SphereAgent: ")
-                spear.log(obs["position"])
-                spear.log(obs["rotation"])
-                spear.log(obs["camera.final_color"].shape, " ", obs["camera.final_color"].dtype)
-                spear.log(reward, " ", done, " ", info)
-        elif config.SIMULATION_CONTROLLER.AGENT == "OpenBotAgent":
-            obs, reward, done, info = env.step(action={"apply_voltage": np.array([1.0, 0.715], dtype=np.float32)})
+                spear.log("    SphereAgent: ")
```

Comment:
`"SphereAgent:"` (no need to indent this, but we should indent the other stuff because then the print statements will be grouped logically under the `SphereAgent:` heading)

Reply:
Updated this